.ci/magician/cmd/vcr_cassette_update.go

```diff
@@ -103,7 +103,7 @@ var vcrCassetteUpdateCmd = &cobra.Command{
 		}
 		ctlr := source.NewController(env["GOPATH"], "hashicorp", env["GITHUB_TOKEN_CLASSIC"], rnr)
 
-		vt, err := vcr.NewTester(env, "ci-vcr-cassettes", "", rnr)
+		vt, err := vcr.NewTester(env, "ci-vcr-cassettes", "", rnr, false)
```

Comment:
I think we can also turn on the async upload for nightly-vcr runs. Or we can enable it later when we confirm this works